dev/scripts/server545_puppet3_compatibility.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+# This script is expected to be run with a PWD of the root of the puppet-server
+# project.
+
+
+# Puppet Server package version (from http://builds.puppetlabs.lan/puppetserver/)
+export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.04.17T0057'
+# Legacy means pre-aio (4.0) in this case
+export PUPPET_LEGACY_VERSION='3.7.5'
+
+  # --post-suite acceptance/suites/post_suite_backwards_compatibility \
+bundle exec beaker \
+  --debug \
+  --root-keys \
+  --no-color \
+  --repo-proxy \
+  --preserve-hosts never \
+  --type aio \
+  --config acceptance/config/beaker/jenkins/redhat7-64m-64a.yaml \
```

Comment:
@jeffmccune I wouldn't be opposed to releasing genconfig as its own publicly-available gem, maybe call it `beaker-genconfig`. I will bring this up with the QE team and let you know if we decide this is a viable option.